proto/kuksa/val/v2/val.proto

```diff
@@ -247,8 +247,9 @@ message BatchActuateResponse {
 }
 
 message ListMetadataRequest {
-  string root   = 1;
-  string filter = 2;
+  // A path which may include wildcards
+  // See https://github.com/eclipse-kuksa/kuksa-databroker/blob/main/doc/wildcard_matching.md
+  string path   = 1;
```

Comment:
Some points of critic:
* With the removed filter one cannot request certain "types" of metadata any longer (like "I just want to get the numeric ids").
* Having just one path (formerly root), it's hard to request metadata in a single request for completely different branches, like `Vehicle.Speed` and `Vehicle.Cabin.Seat.*`. (Or?)

Reply:
Well as of today we do not care about "filter" at all. I have no problem having a filter field if we actually intend to use it, but we do not describe how filter shall/can be used at all right now, like if it going to support the use-cases you mention. But as we seem to be uncertain I put this PR on hold. 

In #105 I instead pointed out that `filter` currently is not used, and that current implementation may change. Like to be on the safe side, do not use wildcards in root, as it may not be supported in the future.